tensorrt_llm/_torch/pyexecutor/self_benchmark.py

```
@@ -0,0 +1,2129 @@
+        )
+        self._identity = self._build_identity()
+        if not self._done:
+            logger.info("Self-benchmark enabled: %s", self.config)
+            initialization_stage = "output_invalidation"
+            try:
+                # Invalidate any stale results from a previous run up front,
+                # so a crash mid-sweep cannot leave a prior run looking valid.
+                self._invalidate_output()
+                initialization_stage = "case_planning"
+                self._cases = self._build_cases()
+            except Exception as exc:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)